packages/paste-core/components/button/src/SecondaryIconButton.tsx

```diff
@@ -40,15 +40,19 @@ const ButtonStyleMapping = {
 };
 
 const SecondaryIconButton = React.forwardRef<HTMLButtonElement, DirectButtonProps>(
-  ({size, buttonState, fullWidth, ...props}, ref) => {
+  ({size, buttonState, fullWidth, pressed, ...props}, ref) => {
+    const toggleStyles = pressed === undefined ? {} : ToggleIconButtonStyles;
+
     // Must spread size styles after button styles
     return (
       <Box
         ref={ref}
+        aria-pressed={pressed}
```

Comment:
suggestion: it is probably worth while throwing some tests around these to check that

1) aria-pressed is set on the element correctly
2) the pressed styles are applied in the pressed state. I notice that you only have one preset pressed button example for VRT too. We probably need test cases for all states.

Random thought, is there anything required about setting aria-pressed to false? It's a string attribute which means it needs to be `aria-pressed="true"` or `aria-pressed="false"`, but we're treating it as a boolean which means it's `aria-pressed="true"` or not present. Does AT do OK with removal of pressed when doing a state change?

Reply:
Oh! I guess the way its rendered, when I set `aria-pressed={false}` it shows up in the dom as `aria-pressed="false"`. I double checked in the website and storybook and it behaves correctly. 

The only time it doesn't add the attribute in the DOM is if it is `aria-pressed={undefined}`.

Reply:
I added tests and confirmed that it works as expected.